docs/pages/access-controls/access-graph/aws-sync.mdx

```diff
@@ -0,0 +1,212 @@
+and resources retrieved from your AWS account, crafting a 
+graphical representation thereof.
+
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v14.3.9/v15.2.0 or later.
+- For self-hosted clusters, an updated `license.pem` with Teleport Policy enabled.
+- Docker version v(=docker.version=) or later.
+- A running Teleport Access Graph node v1.7.0 or later.
+- The node running the Access Graph service must be reachable
+from Teleport Auth Service and Discovery Service. Check 
+[Access Graph page](../access-graph.mdx) for details on
+how to setup Teleport Access Graph.
+
+## Step 1/2. Configure Discovery Service (Self-hosted only)
+
+<Notice type="warning">
+
+If you have a Teleport Cloud cluster, please disregard
+this step, as Teleport Cloud already operates a properly configured 
+Discovery Service within your cluster.
+</Notice>
+
+To activate the Teleport Discovery Service,
+add the provided snippet to your Auth Service configuration. 
+This service monitors dynamic `discovery_config` resources
+that are set up with the `discovery_group` matching 
+<Var name="access-graph-disc" />.
+
+
+```code
+discovery_service:
+  enabled: true
+  discovery_group: <Var name="access-graph-disc" />
+
+```
+
+Notice that if you already operate a Discovery Service within your cluster,
+it's possible to reuse it as long as the following requirements are met:
+
+- On step 2, you match the `discovery_group` with the existing Discovery Service's
+`discovery_group`.
+- Access Graph service is reachable from the machine where Discovery Service runs.
+
+## Step 2/2. Set up Access Graph AWS Sync
+
+To initiate the setup wizard for configuring AWS Sync, access the Teleport UI,
+navigate to the Management tab, and choose the Access Graph option within the
+Permission Management section.
+
+If both Teleport and Access Graph support AWS sync, you'll notice a new button
+adjacent to the Access Graph navigation bar labeled `Set up AWS sync with
+Access Graph`.
+
+You'll be
+prompted to create a new Teleport AWS integration if you haven't configured
+one already. Alternatively, you can opt for a previously established integration.
+
+Upon selecting or creating the integration, you'll be instructed to execute a
+bash script within your AWS Cloud Shell to configure the necessary permissions.
+
+<Details title="List of IAM Policies required for AWS Sync" closed>
+
+As part of the setup process, you'll be prompted to execute a bash script
+within your AWS Cloud Shell. This script's purpose is to generate an IAM
+Policy within your AWS account and link it to the AWS Role designated for
+your Teleport AWS Integration. The policy is designed with a set of read-only
+actions, enabling Teleport to access and retrieve information from resources
+within your AWS Account.
+
+The IAM policy includes the following directives:
+
+
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        // EC2 IAM
```

Comment:
JSON doesn't support comments. I'd either remove them to keep valid JSON or not use JSON here.